modules/react/layout/stories/Box.stories.mdx

```diff
@@ -110,321 +33,18 @@ noticeable transformation is that `Box` renders as a link. But also notice that
-
----
-
-#### Position Standard Props
-
-Position standard props work the same as position logical props, but have no bidirectional support.
-Bidirectional support will only affect `left` and `right`. Below is a props table for the standard
-props:
-
-<ArgsTable of={BoxPositionStandard} />
-
-#### Example
-
-<ExampleCodeBlock code={Position} />
-
----
-
-### Space
-
-Space style props allow you to adjust the margin and padding of components. There are two types of
-space props: `SpaceLogicalProps` and `SpaceStandardProps`.
-
-#### Space Logical Props
-
-Space logical props are based on
-[CSS Logical Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties/Basic_concepts).
-They support bidirectionality (LTR and RTL) and will automatically flip the value when a change in
-direction is detected. For example: `marginInlineStart="16px"` would set `margin-left: 16px;` for
-LTR and switch to `margin-right: 16px;` for RTL. You'll want to opt for these properties over
-standard space props, such as `marginLeft` and `marginRight` when you need bidirectional styles.
-
-Here's a quick reference for how these props map to LTR and RTL styles:
-
-| Logical Prop         | LTR Style       | RTL Style       |
-| -------------------- | --------------- | --------------- |
-| `marginInlineStart`  | `margin-left`   | `margin-right`  |
-| `marginInlineEnd`    | `margin-right`  | `margin-left`   |
-| `paddingInlineStart` | `padding-left`  | `padding-right` |
-| `paddingInlineEnd`   | `padding-right` | `padding-left`  |
-
-And below is a props table for the logical props:
-
-<ArgsTable of={BoxSpaceLogical} />
-
----
-
-#### Space Standard Props
-
-Space standard props work the same as logical space props, but have no bidirectional support.
-Bidirectional support will only affect `marginLeft`, `marginRight`, `paddingLeft`, and
-`paddingRight`. Below is a props table for the standard props:
-
-<ArgsTable of={BoxSpaceStandard} />
-
----
-
-#### Example
-
-<ExampleCodeBlock code={Space} />
-
-### Text
-
-Text style props allow you to modify text using props such as `textDecoration` or `textTransform` or
-set line styles by `lineHeight` or `letterSpacing`, etc.
-
-#### Example
-
-<ExampleCodeBlock code={Text} />
-
----
-
-## Exhaustive Style Prop List
-
-Below is an exhaustive list of `Box` style props divided by category.
-
-### Border Props
-
-<ArgsTable of={BoxBorder} />
-
-### Color Props
-
-<ArgsTable of={BoxColor} />
-
-### Depth Props
-
-<ArgsTable of={BoxDepth} />
-
-### FlexItem Props
-
-<ArgsTable of={BoxFlexItem} />
-
-### Layout Props
-
-<ArgsTable of={BoxLayout} />
-
-### Position Props
-
-<ArgsTable of={BoxPosition} />
-
-### Space Props
-
-<ArgsTable of={BoxSpace} />
+### Style Props
+
+`Box` exposes [style props](/getting-started/for-developers/resources/style-props) that allow you to modify styles in an ergonomic way across components. To learn more about individual sets of `Box` style props, consult the lists below.
```

Comment:
The `/getting-started/for-developers/resources/style-props` link will work in Storybook as long as you add a trailing slash at the end (`.../style-props/`) and add the following entry to `routes.js`:

```
'/getting-started/for-developers/resources/style-props/': 'welcome-dev-docs-style-props--background-example',
```

The links with hashes at the end which you've included throughout this PR (e.g., `/getting-started/for-developers/resources/style-props/#background`) **won't** work in Storybook, but they **will** work on the Canvas Site. The hash doesn't work for our `stories.mdx` files in Storybook (such as your [STYLE_PROPS.stories.mdx](https://5d854c26ba934e0020f5e98a-bkpbzhpejs.chromatic.com/?path=/docs/welcome-dev-docs-style-props--background-example)) because their headings aren't given `id`s, whereas non-story MDXes (such as the [8.0-UPGRADE-GUIDE.mdx](https://5d854c26ba934e0020f5e98a-bkpbzhpejs.chromatic.com/?path=/docs/welcome-upgrade-guides-v8-0--page)) do have `id`s assigned to headings.

Reply:
Got. Thanks for the context! It looks like the hashed links won't take them to the section, but it will at least get them to the correct page in Storybook. I'm okay with that if it works better for the site.